src/components/Chat/ChatMessage.tsx

```diff
@@ -385,10 +396,7 @@ export const ChatMessage: FC<Props> = memo(
                         <button
                           className="button button-primary"
                           onClick={handleEditMessage}
-                          disabled={
-                            messageContent.trim().length <= 0 &&
-                            newEditableAttachments.length <= 0
-                          }
+                          disabled={isSubmitAllowed}
```

Comment:
this should be checked in `handleEditMessage` to avoid executing on `Enter` click